scripts/lib/CIME/SystemTests/pgn.py

```diff
@@ -44,8 +44,7 @@
 FCLD_NC = 'cam.h0.cloud.nc'
 INIT_COND_FILE_TEMPLATE = \
     "SMS_Ly5.ne4_ne4.FC5AV1C-04P2.eos_intel.ne45y.{}.{}.0002-{:02d}-01-00000.nc"
```

Comment:
@singhbalwinder why run FC5AVIC-04P2?   Does that compset still exist?

Reply:
Actually I'm not sure what this TEMPLATE variable is doing.  "eos" is also a machine that doesn't exist anymore.

Reply:
@rljacob : The TEMPLATE variable is used for generating the name of the initial condition files. The initial conditions are generated using FC5AVIC-04P2 on eos machine. That is why you see both the old compset name and the machine name in the TEMPLATE variable. Since this is just an initial condition file, the compset used doesn't matter. More specifically, it should not make a difference for these particular tests. We can still run these tests using new v2 compsets.